.pylintrc

```diff
@@ -54,6 +54,7 @@ ignore-imports=yes
 #    missing-docstring
 #    no-member
 #    no-self-use
+#    relative-import
```

Comment:
Done, I'm not sure what changed but I can push again. I added this b/c I wasn't able to push without it.